### PR TITLE
Feature/map 175/add swiper slider

### PR DESCRIPTION
**Pull request by obozhko-dev**
Added swiper caurosel for images https://swiperjs.com/react in Big Popup

image

### CODE REVIEW

**Review by Yojuks — APPROVED** [review]

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -42,17 +43,17 @@
...
-        "ts": "never",
-        "tsx": "never"
-      }
     ]
+    // "import/extensions": [
+    //   "error",
+    //   "always",
+    //   {
+    //     "js": "never",
+    //     "jsx": "never",
```
Comment (Yojuks) [inline frontend/.eslintrc.json:52]: delete

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/src/components/PointPopup/LocationImageCarousel/LocationImageCarousel.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// Import Swiper React components
+/* eslint-disable import/no-unresolved */
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { CardMedia } from '@mui/material';
+import React from 'react';
+import { Navigation, Pagination } from 'swiper';
+import './styles.css';
+
+// Import Swiper styles
```
Comment (Yojuks) [inline frontend/src/components/PointPopup/LocationImageCarousel/LocationImageCarousel.tsx:9]: delete

File: frontend/src/components/PointPopup/LocationImageCarousel/LocationImageCarousel.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+import { v4 } from 'uuid';
+// import { CardMediaCarousel } from 'components/design/StyledProfile copy';
```
Comment (Yojuks) [inline frontend/src/components/PointPopup/LocationImageCarousel/LocationImageCarousel.tsx:16]: delete

File: frontend/src/components/PointPopup/PointPopup.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {
   Box,
   Card,
   CardContent,
-  CardMedia,
+  // CardMedia,
```
Comment (Yojuks) [inline frontend/src/components/PointPopup/PointPopup.tsx:7]: delete

File: frontend/src/components/PointPopup/PointPopup.tsx [hunk 5]
```diff
@@ -116,11 +117,15 @@ const PointPopup = () => {
       ) : (
         <Box>
           <Card>
-            <CardMedia
+            {/* <CardMedia
               sx={{ p: 2 }}
               component="img"
               image={arrayPhotos[0]}
               alt={locationName}
```
Comment (Yojuks) [inline frontend/src/components/PointPopup/PointPopup.tsx:124]: delete

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/.eslintrc.json (hunk 1 above)
Comment (obozhko-dev) [reply frontend/.eslintrc.json:52]: done

**Pull request closed without merging**